### PR TITLE
Fixes #18540: Healthcheck checks happen to often 

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/18540

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala [hunk 1]
```diff
@@ -577,11 +577,11 @@ object RudderConfig extends Loggable {
     Duration.fromScala(scala.concurrent.duration.Duration(config.getString("metrics.healthcheck.scheduler.period")))
   } catch {
     case ex: ConfigException =>
-      ApplicationLogger.info("Property 'metrics.healthcheck.scheduler.period' is missing or empty in rudder.configFile. Default to 5mins.")
-      5.minutes
+      ApplicationLogger.info("Property 'metrics.healthcheck.scheduler.period' is missing or empty in rudder.configFile. Default to 6 hours.")
+      20.seconds
```
Comment (fanf) [inline webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala:581]: 6 hours ?

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/healthcheck/HealthcheckNotificationService.scala [hunk 2]
```diff
@@ -50,10 +50,10 @@ class HealthcheckNotificationService(
     v <- Ref.make[List[HealthcheckResult]](List.empty)
   } yield v).runNow
 
-  private[this] def reloadCache(ref: Ref[List[HealthcheckResult]]): UIO[Unit] = {
+  def reloadCache(ref: Ref[List[HealthcheckResult]] = healthcheckCache): UIO[Unit] = {
```
Comment (fanf) [inline webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/healthcheck/HealthcheckNotificationService.scala:53]: why that change? 
Default parameters are a code smell, they tend to bring a lot of complicated problem on the long term. 
If you need a way to say "now, do init your internal things", it's better to have a dedicated `init` method. That way, it's also easier to understand you class state machine (entry points, transition, etc).

File: webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala [hunk 3]
```diff
@@ -536,6 +536,9 @@ class Boot extends Loggable {
     }
   }
 
+  // Run a health check
+  RudderConfig.healthcheckNotificationService.reloadCache()
```
Comment (fanf) [inline webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala:540]: I would prefer a `startService` here.

Comment (ElaadF) [reply webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala:540]: a duplicated `healthcheckNotificationService` used only at boot ?

Comment (fanf) [reply webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala:540]: no, just the method: `RudderConfig.healthcheckNotificationService.startService`. Or `start` or `init`

**Review by fanf — APPROVED** [review]

**Pull request merged**